hs_core/management/commands/delete_model_aggregation_folder_from_irods.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import os
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from hs_model_program.models import ModelProgramResource
+from hs_modelinstance.models import ModelInstanceResource
+from hs_modflow_modelinstance.models import MODFLOWModelInstanceResource
+from hs_swat_modelinstance.models import SWATModelInstanceResource
+
+
+class Command(BaseCommand):
+    """This command needs to be run to cleanup model aggregation folder from iRODS when testing model migration commands
+    NOTE: This command MUST not be used in production
+    """
+
+    help = "Save the id of the linked model program resource in model instance resource as part of preparing model" \
+           "instance resource migration to composite resource"
+
+    def add_arguments(self, parser):
+
+        parser.add_argument('type', type=str,   help='limit to resources of a particular model resource type')
+        # a list of resource id's for which aggregation folder needs to be deleted.
+        parser.add_argument('resource_ids', nargs='*', type=str)
+
+    def process_resource(self, model_res, resource_type, logger):
+        istorage = model_res.get_irods_storage()
+        model_res_type_folder_mapping = {"ModelProgram": "model-program", "ModelInstance": "model-instance",
+                                         "MODFLOWModelInstance": "modflow-instance",
+                                         "SWATModelInstance": "swat-instance"}
+        if not istorage.exists(model_res.root_path):
+            err_msg = ">> Resource (ID:{}) doesn't exist in iRODS"
+            self.stdout.write(self.style.ERROR(err_msg))
+            self.stdout.flush()
```

Comment:
you can simplify it with ```print(message, flush=True)```

Reply:
I am using `self.style.ERROR(err_msg))`  to easily see the error messages (displayed in red text) when testing. That's the reason I didn't use `print`